arches/management/commands/updateproject.py

```diff
@@ -103,7 +103,7 @@ def update_to_v7_6(self):
             print("Copying {} to project root directory".format(dotfile))
             shutil.copy2(os.path.join(settings.ROOT_DIR, "install", "arches-templates", dotfile), os.path.join(settings.APP_ROOT, '..'))
 
-        for file in [".yarnrc", "yarn.lock"]:
+        for file in [".yarnrc", "yarn.lock", ".eslintignore", ".eslintrc.js",]:
             if os.path.exists(os.path.join(settings.APP_ROOT, file)):
                 print("Ensuring {} is deleted from project sub-directory".format(file))
                 os.remove(os.path.join(settings.APP_ROOT, file))
```

Comment:
`updateproject` took a dive for me because the last line still expects the webpack template folder under install/, so if we're deleting that in this PR let's update the command, or maybe even better do it in #10758.